qt_ui/windows/mission/QMissionPlanning.py

```diff
@@ -15,9 +15,7 @@ class QMissionPlanning(QDialog):
     def __init__(self, game: Game):
         super(QMissionPlanning, self).__init__()
         self.game = game
-        self.setWindowFlags(Qt.WindowStaysOnTopHint)
```

Comment:
I think this line should stay, otherwise the Mission Planner window will drop behind the DCS Liberation Windows. With todays screen size there should be enough space to arrange the windows so the map can be seen.